plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryCaseInsensitiveMapping.java

```diff
@@ -226,6 +226,25 @@ public void testTableNameClash()
         }
     }
 
+    @Test
+    public void testCreateSchema()
+    {
+        String schemaName = "Test_Create_Case_Sensitive_" + randomTableSuffix();
+        assertUpdate("CREATE SCHEMA " + schemaName.toLowerCase(ENGLISH));
+        assertQuery(format("SELECT schema_name FROM information_schema.schemata WHERE schema_name = '%s'", schemaName.toLowerCase(ENGLISH)), format("VALUES '%s'", schemaName.toLowerCase(ENGLISH)));
+        assertUpdate("DROP SCHEMA " + schemaName.toLowerCase(ENGLISH));
```

Comment:
@hashhar does this test have sense within `CaseInsensitiveMapping` class since all occurrences can be extracted
`String lowerCase = schemaName.toLowerCase(ENGLISH);` 
 and used through out the method?